pkg/reconciler/apiresource/negotiation.go

```diff
@@ -552,21 +552,29 @@ func (c *Controller) ensureAPIResourceCompatibility(ctx context.Context, cluster
 		})
 	}
 	if negotiatedAPIResource == nil {
-		existing, err := c.kcpClient.ApiresourceV1alpha1().NegotiatedAPIResources().Create(ctx, newNegotiatedAPIResource, metav1.CreateOptions{})
```

Comment:
does this commit belong into the PR?

Reply:
Not really, but it makes the test pass *way* faster, so ... :)

Reply:
Commit removed